terraform/main.tf

```diff
@@ -165,7 +165,6 @@ module "batch_data_lake_writer" {
   suffix                   = local.name_suffix
   tags                     = local.batch_tags
   use_ephemeral_storage    = true
-  launch_type              = "EC2"
```

Comment:
Minor thing but it may be good to keep the argument and set it to "SPOT" so we don't have to figure out how it's set again if needed.

Reply:
Good idea.